packages/core/phase1/phase1.ts

```diff
@@ -61,9 +61,7 @@ const phase1 = async (
     resultType = Phase1ResultType.ignored
   } else {
     const exceptions = generateExceptions(enrichedHearingOutcome)
-    exceptions.forEach(({ code, path }) => {
-      addExceptionsToAho(enrichedHearingOutcome as AnnotatedHearingOutcome, code, path)
-    })
+    addExceptionsToAho(enrichedHearingOutcome as AnnotatedHearingOutcome, exceptions)
```

Comment:
This looks better, though it made me notice the casting in this part of the code which is normally a bad sign - is there a reason it's there?

Reply:
I removed it and was able to build and run unit tests so seems like it's redundant.